client/src/routes/api/login/+server.ts

```diff
@@ -0,0 +1,39 @@
+import backendConfig from '$server/backend.config';
+import serverConfig from '$server/server.config';
+import type { LoginRequestData } from '$shared/api/login';
+
+/** @type {import('@sveltejs/kit').RequestHandler} */

```

Comment:
Comment types do not belong in TS files. For us to proceed, there needs to be strong feelings of disgust in you against these, please develop them. Search the docs, there is a switch somewhere in them to switch the example typings to something that is not a crime against humanity.